packages/mural-picker/src/components/theme.ts

```diff
@@ -2,31 +2,90 @@ import { createMuiTheme, ThemeOptions } from '@material-ui/core/styles';
 
 export type Preset = 'light' | 'dark';
 
-export const THEMES: Record<Preset, ThemeOptions> = {
-  light: {
-    palette: {
-      type: 'light',
-      text: { primary: '#585858' },
-      primary: {
-        main: '#FF0066',
+const base = createMuiTheme({
+  shape: {
+    borderRadius: 8,
+  },
+  typography: {
+    fontFamily: 'Proxima Nova',
+    button: {
+      lineHeight: 1.2,
+      textTransform: 'none',
+    },
+  },
+  overrides: {
+    // @ts-ignore
+    MuiAutocomplete: {
+      inputRoot: {
+        '&[class*=MuiOutlinedInput-root]': {
+          padding: '0 6px',
+        },
+        '& fieldset': {
+          borderWidth: 2,
+        },
+      },
+      listbox: {
+        border: '1px solid lightgray',
+      },
+    },
+    MuiFormLabel: {
+      root: {
+        fontWeight: 600,
+      },
+    },
+    MuiCardActionArea: {
+      root: {
+        padding: '0.8rem',
       },
     },
-    typography: {
-      fontFamily: 'Proxima Nova',
+    MuiButton: {
+      root: {
+        height: '2.8em',
+      },
```

Comment:
question: Should the theme define a focus style for the buttons? Right now I don't see any visual indicator of the focus.

Reply:
I've updated the code to have a proper focus state now.